charts/iac/crossplane-aws-iam/templates/policy.yaml

```diff
@@ -14,8 +14,24 @@ spec:
     {{- include "common-gitops.crossplane.awsDeletionPolicy" (dict "root" $root "name" $name "kind" $kind) | nindent 2 }}
   forProvider:
     name: {{ include "common-gitops.names.itemFullname" (dict "root" $root "name" $name "override" (.forProvider).name) }}
+    {{ with (.forProvider).document -}}
+    document: |
+      {
+        "Version": "{{ .Version | default "2012-10-17" }}",
+        "Statement": [
```

Comment:
array needs to be created only when there are more than one item. With just one item in the array, AWS starts fighting with crossplane

Reply:
With just one item in the array, AWS converts array into the string. Crossplane detects it as a drift and corrects it. This right results in very high api calls rate from crossplane pods